projects/media-viewer/src/lib/toolbar/toolbar-button-visibility.service.ts

```diff
@@ -72,13 +74,19 @@ export const defaultImageOptions = {
   showZoom: true,
   showRotate: true,
   showGrabNDragButton: true,
-  showCommentSummary: true
+  showCommentSummary: true,
+  showRedact: true
 };
 
 /**
  * Default toolbar state for unsupported media
  */
 export const defaultUnsupportedOptions = {
   showDownload: true,
-  showCommentSummary: false
+  showCommentSummary: false,
+  showDrawButton: false,
```

Comment:
you shouldn't need to add any false options here, because everything is false by default, and you just set things to true if you want them to appear